readme.md

```diff
@@ -2,6 +2,7 @@
 
 This repository contains the following:
 * **Microsoft Azure IoT Hub device SDK for C** to connect devices running C code to Azure IoT Hub
+* **Microsoft Azure Iot Hub Device Provisioning Service client SDK** for enrolling devices with Device Provisioning Services
```

Comment:
Can we add: for enrolling devices with Device Provisioning Services and managing enrollment lists.